webapp/channels/src/components/suggestion/suggestion_list.tsx

```diff
@@ -164,8 +174,8 @@ export default class SuggestionList extends React.PureComponent {
         }
     };
 
-    getComputedCssProperty(element, property) {
-        return parseInt(getComputedStyle(element)[property], 10);
+    getComputedCssProperty(element: Element | Text, property: string) {
+        return parseInt(getComputedStyle(element as HTMLElement).getPropertyValue(property), 10);
```

Comment:
Debugging for a bit myself, it looks like the issue with scrolling is actually this line. `getComputedStyle(element)[property]` returns the string `0px` for unset properties while `getComputedStyle(element as HTMLElement).getPropertyValue(property)` returns an empty string which is making this return `NaN` in some cases which screws up the logic in `scrollToItem`.

If you add a `|| '0'` to this, it seems to work. I even tested re-adding the `instanceof HTMLElement` check as well, and that seemed to work, but I didn't test it thoroughly, so I'd be fine going either way with it